lib/grape/jsonapi/exposer.rb

```diff
@@ -21,6 +21,16 @@ def self.field(instance, field)
         end
       end
 
+      def self.one_level_deep?(field)
+        field_exist = field_exists?(field)
+        lambda do |instance, options|
+          if field_exist.call(instance, options)
+            return true if options.opts_hash.dig(:attr_path).count(:included) == 1
```

Comment:
I think you can get rid of the `return true if` here.

Reply:
Nm I see that's an `end`, not an `else` below.

Reply:
```
depth = options.opts_hash.dig(:attr_path).count(:included)
field_exist.call(instance, options) && depth == 1
```